File: benchmarks/runtime/mapping_churn.js
let n = 100000;
let i = 0;
let sum = 0;
while (i < n) {
  const m = new Map([[i, i + 1], [i + 100, i + 2]]);
  sum += m.get(i);
  i += 1;
}
if (sum <= 0) process.exit(1);
console.log(sum);

File: benchmarks/runtime/mapping_churn.lua
local n = 100000
local i = 0
local sum = 0
while i < n do
  local m = {}
  m[i] = i + 1
  m[i + 100] = i + 2
  sum = sum + m[i]
  i = i + 1
end
if sum <= 0 then os.exit(1) end
print(sum)
